### DISCUSSION TITLE
Don't make things more complicated than they need to be - if in doubt ask!!!!

### DISCUSSION THREAD

**Discussion by metasean** [USU-ITLS-x265-23-summer/class-repo#12] [Announcements]
Remember [Task 2](https://github.com/USU-ITLS-x265-23-summer/class-repo/blob/main/Tasks/Task-2.md)!    There's a reason that it's one of the first assignments! 😉 

If you are unfamiliar with xkcd comics, read the whole comic, then hover over it for extra bit:
https://xkcd.com/530/ 

If it sounds like a task should take less than 5 minutes and you've already spent 15 minutes on it, that's a huge sign that you should take a step back, tell your rubber ducky the instructions and problem in detail, and if that fails, come here and ask!!!!

While I am encouraging you to limit your use of AI, but I'm not prohibiting it.  In fact, once you transition to Codespaces, there are some pretty good AI extensions.  For where you are right now, chatpgt ($$$) and it's ilk _may_ be useful —remember to always take their response with several grains of salt— but they are getting better by the day.  [codepal.ai](https://codepal.ai/) looks like it's geared specifically toward novice developers (e.g. [Why would I use a Pull Request?](https://codepal.ai/rubber-duck/query/4P591Zo2/pull-requests-collaborative-development)).  It is free, but it is also painfully slow!

**Comment by kassyoveson** [comment]
there is also www.bard.google.com. It is also free. It is in the beta testing, but I have found it useful.
